webpack/assets/javascripts/foreman_navigation.js

```diff
@@ -1,7 +1,10 @@
 import $ from 'jquery';
+import { push } from 'connected-react-router';
 import store from './react_app/redux';
 import * as LayoutActions from './react_app/components/Layout/LayoutActions';
 
+export const pushUrl = url => store.dispatch(push(url));
```

Comment:
Should we support operations like `updateUrlQueryParams`?

Reply:
bump?

Reply:
What would that do?

Reply:
It would update the `query-params` without changing the URL.
It might be useful for pagination and filtering.